app/models/user.rb

```diff
@@ -112,8 +111,7 @@ def original_reset_password_token
   def regenerate_delete_account_token
     @delete_account_token_original = SecureRandom.urlsafe_base64
     update!(
-      delete_account_token_digest:
-        Digest::SHA2.hexdigest(@delete_account_token_original)
+      delete_account_token_digest: Digest::SHA2.hexdigest(@delete_account_token_original)
```

Comment:
Incorrect formatting, please use the [recommended config](https://docs.pupilfirst.com/developers/development_setup#code-formatting)